### ISSUE TITLE
Modal: Change header to H1

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#29215]
## Description

When the component library "alert" modal is visible, an `h1` header should be present to fulfill a11y scans.

To test:
- Go to https://design.va.gov/storybook/iframe.html?id=components-modal--default&viewMode=story
- Open the modal
- Run axe DevTools

Note: the page itself doesn't have an H1, but even if it did, it gets ignored by the scan.

<details><summary>a11y error</summary>

![Screen Shot 2021-08-25 at 11 20 16 AM](https://user-images.githubusercontent.com/136959/130827782-54e14caf-792a-4b4d-b1ab-3a64349adcad.png)</details>

## Tasks

- [ ] Change modal `h3` to an `h1` & adjust header size
- [ ] Add unit/a11y tests

## Definition of done

- [ ] All tasks done
- [ ] All tests passing

**Comment by Mottie** [comment]
Also, including a "main landmark" also appears to be a related issue - this announcement modal _does_ have an `H1`.

<details><summary>Announcement modal</summary>


![Screen Shot 2021-08-25 at 11 34 17 AM](https://user-images.githubusercontent.com/136959/130830100-a7635cfa-4b6c-4c98-bdbb-29d389871240.png)</details>

**Comment by k80bowman** [comment]
Once this work is done, please work with @CourtneyChambers77 to verify that the issue is resolved.

**Comment by CourtneyChambers77** [comment]
In my evaluation I am seeing the header as  H3 please see below 
<img width="695" alt="image" src="https://user-images.githubusercontent.com/95867638/148559532-8cd25a5b-9807-415f-863a-5852073acdb6.png">

**Comment by bkjohnson** [comment]
What URL was that?

**Comment by bkjohnson** [comment]
Here's the URL that I think would show the correct heading level: https://60f9b557105290003b387cd5-mimoztzxmy.chromatic.com/iframe.html?id=components-modal--default&viewMode=story
